aiohttp/cookiejar.py

```diff
@@ -48,15 +47,16 @@ class CookieJar(AbstractCookieJar):
 
     DATE_YEAR_RE = re.compile(r"(\d{2,4})")
 
-    MAX_TIME = 2051215261.0  # so far in future (2035-01-01)
+    MAX_TIME: datetime.datetime = datetime.datetime.max.replace(
```

Comment:
Don’t use variable annotations, this is the 3.5 syntax error you see. Put type hints in a comment:

```python 
MAX_TIME = datetime.datetime.max.replace(
    tzinfo=datetime.timezone.utc)  # type: datetime.datetime
```

I don’t think a type hint is needed here however; dropping it is fine:

```suggestion
    MAX_TIME = datetime.datetime.max.replace(
```